### PR TITLE
Adds GPU Dockerfile, convenience Makefile, resolves #1

### LINKED ISSUE #1: Provide nvidia-docker based Dockerfile
The Dockerfile we provide right now is cpu based only.

We used that for porting and toying around with the example.

We should provide an additional gpu Dockerfile as an example and to get folks started.

Take inspiration e.g. from my https://github.com/daniel-j-h/efficientnet

### PR DESCRIPTION
**Pull request by daniel-j-h**
Readme preview

https://github.com/moabitcoin/ig65m-pytorch/tree/issues/1#ig-65m-pytorch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,47 +15,72 @@ We also [provide](https://github.com/moabitcoin/ig65m-pytorch/releases) converte
 |  R(2+1)D_34   | IG-65M + None       | 32x112x112 | [*r2plus1d_34_clip32_ig65m_from_scratch_449a7af9.pth*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ig65m_from_scratch_449a7af9.pth)                                               | [*r2plus1d_34_clip32_ig65m_from_scratch_e304d648.pb*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ig65m_from_scratch_e304d648.pb)                                            |
 |  R(2+1)D_34   | IG-65M + Kinetics  | 32x112x112 | [*r2plus1d_34_clip32_ft_kinetics_from_ig65m_ade133f1.pth*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ft_kinetics_from_ig65m_ade133f1.pth) | [*r2plus1d_34_clip32_ft_kinetics_from_ig65m_10f4c3bf.pb*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ft_kinetics_from_ig65m_10f4c3bf.pb)  |
```
Comment (sandhawalia) [inline README.md]: Should just add another column with caffe model .pkl file ?

File: README.md [hunk 2]
```diff
@@ -23,47 +15,72 @@ We also [provide](https://github.com/moabitcoin/ig65m-pytorch/releases) converte
...
 
-positional arguments:
-  pkl                  .pkl file to read the R(2+1)D 34 layer weights from
-  out                  prefix to save converted R(2+1)D 34 layer weights to
+By default we build and run the CPU Docker images; for GPUs run:
 
-optional arguments:
-  -h, --help           show this help message and exit
-  --frames {8,32}      clip frames for video model
-  --classes {400,487}  classes in last layer
```
Comment (sandhawalia) [inline README.md:39]: I think we should explain what each of the numbers mean. 400 for kinetics, 487 for T=8 and 259 for T=32 ?

**Review by sandhawalia — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (daniel-j-h) [reply README.md]: Moved

File: README.md (hunk 2 above)
Comment (daniel-j-h) [reply README.md:39]: Yes, adding

**Pull request merged**